sync/sync-impl/src/main/java/com/duckduckgo/sync/impl/ui/SyncConnectViewModel.kt

```diff
@@ -91,9 +144,13 @@ class SyncConnectViewModel @Inject constructor(
         }
```

Comment:
If I'm not wrong `onShowQRCodeClicked()` doesn't exist anymore, so everything that happens from here can be removed. For example: `ShowQRCodeActivity`

Reply:
good catch, remove them all!